patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -837,13 +837,18 @@
 +     */
 +    public boolean canPlaceTorchOnTop(IBlockState state, IBlockAccess world, BlockPos pos)
 +    {
-+        if (state.func_185896_q() || state.func_193401_d(world, pos, EnumFacing.UP) == BlockFaceShape.SOLID)
++        if (this == Blocks.field_185775_db || this == Blocks.field_150428_aP)
 +        {
-+            return this != Blocks.field_185775_db && this != Blocks.field_150428_aP;
++            return false;
```

Comment:
This line will never be executed

Reply:
@mvlabat No, this is removing the `return this...` line and adding `return false` instead.

Reply:
Oh, I'm sorry. Is it some kind of github UI update? The line is still highlighted with green, I was totally confused by it